.github/workflows/ci.yml

```diff
@@ -55,6 +58,7 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.PYTHON.VERSION }}
+          allow-prereleases: true
```

Comment:
We prefer explicitly specifying `3.13-dev`, not doing `allow-prereleases`, and then switching on release day.

Reply:
Updated!